### PR TITLE
Support reverse proxy based and local setups

### PR DESCRIPTION
**Pull request by Andrew-Folga**

### CODE REVIEW

File: proxy.conf
Comment (alanplatt) [inline proxy.conf:1]: Why do we have two different proxy configs?

File: nginx.local.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    server {
+        listen 80;
+
+        # Proxy requests to the webapp
+        location /eligibility-checker/ {
+            proxy_pass http://ffc-grants-eligibility-checker:3000/;
+        }
+
+        # Proxy static files directly to the app
```
Comment (alanplatt) [inline nginx.local.conf]: Is this going to work in dev?

File: proxy.conf
Comment (swdpcomputing) [reply proxy.conf:1]: See Readme

**Review by swdpcomputing — DISMISSED** [review]
Approving pending passing review from other parties.

File: package.json
Comment (swdpcomputing) [inline package.json:1]: Requires version bump.

File: proxy.conf
Comment (alanplatt) [reply proxy.conf:1]: I meant nginx :-P

**Review by alanplatt — DISMISSED** [review]

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**